### PR TITLE
Update to SSD1331 driver

### PR DESCRIPTION
**Pull request by kattni**
Resolves issue with display not functioning properly. Tested successfully with SSD1331 display. Thank you to @deshipu for figuring this out!

### CODE REVIEW

File: adafruit_rgb_display/ssd1331.py [hunk 1]
```diff
@@ -37,23 +42,31 @@ class SSD1331(DisplaySPI):
...
+      display = ssd1331.SSD1331(spi, cs=digitalio.DigitalInOut(board.GPIO0),
+                                  dc=digitalio.DigitalInOut(board.GPIO15),
+                                  rst=digitalio.DigitalInOut(board.GPIO16))
+
+      display.fill(0x7521)
+      display.pixel(32, 32, 0)
+
+    .. code-block:: python
+
+      from machine import Pin, HSPI
```
Comment (tannewt) [inline adafruit_rgb_display/ssd1331.py]: This is for micropython and should be removed. It should exist in the micropython driver.

File: adafruit_rgb_display/ssd1331.py [hunk 2]
```diff
@@ -62,31 +75,42 @@ class SSD1331(DisplaySPI):
...
         (_DISPLAYON, b''),
     )
     _ENCODE_PIXEL = ">H"
     _ENCODE_POS = ">BB"
 
+    # pylint: disable-msg=useless-super-delegation, too-many-arguments
     def __init__(self, spi, dc, cs, rst=None, width=96, height=64):
         super().__init__(spi, dc, cs, rst, width, height)
+
+    # pylint: disable-msg=no-member
```
Comment (tannewt) [inline adafruit_rgb_display/ssd1331.py]: Please add a comment why each of these lint exemptions are needed.

Comment (deshipu) [reply adafruit_rgb_display/ssd1331.py]: We need the `__init__` here to provide default values, so it's not really useless.

Comment (tannewt) [reply adafruit_rgb_display/ssd1331.py]: Ok, thats what I was wondering.

**Pull request merged**